Neos.ContentRepository.Core/Classes/Feature/NodeMove/Event/NodeAggregateWasMoved.php

```diff
@@ -95,16 +76,43 @@ public function createCopyForContentStream(ContentStreamId $targetContentStreamI
         return new self(
             $targetContentStreamId,
             $this->nodeAggregateId,
-            $this->nodeMoveMappings,
+            $this->newParentNodeAggregateId,
+            $this->succeedingSiblingsForCoverage,
         );
     }
 
     public static function fromArray(array $values): self
     {
+        if (array_key_exists('nodeMoveMappings', $values)) {
+            $newParentNodeAggregateId = null;
+            $succeedingSiblings = [];
+            foreach ($values['nodeMoveMappings'] as $nodeMoveMapping) {
+                // we don't care about origins anymore
+                foreach ($nodeMoveMapping['newLocations'] as $newLocation) {
+                    if (array_key_exists('newParent', $newLocation)) {
+                        $newParentNodeAggregateId = NodeAggregateId::fromString($newLocation['newParent']);
+                    }
+                    $succeedingSiblings[] = new InterdimensionalSibling(
+                        DimensionSpacePoint::fromArray($newLocation['coveredDimensionSpacePoint']),
+                        ($newLocation['newSucceedingSibling'] ?? null)
+                            ? NodeAggregateId::fromString($newLocation['newSucceedingSibling'])
+                            : null
+                    );
+                }
+            }
+            $succeedingSiblingsForCoverage = new InterdimensionalSiblings(...$succeedingSiblings);
+        } else {
+            $newParentNodeAggregateId = $values['newParentNodeAggregateId'] === null
+                ? null
+                : NodeAggregateId::fromString($values['newParentNodeAggregateId']);
+            $succeedingSiblingsForCoverage = InterdimensionalSiblings::fromArray($values['succeedingSiblingsForCoverage']);
+        }
```

Comment:
This is the "migration" right?
In other places we (or rather @mhsdesign) decided to go for an event migration instead.

I'm not really decided on that matter but my tendency is to go for an event migration until we have tagged a first RC
In any case we should probably agree on a common strategy for these cases

Reply:
I'd be fine with a migration as well, I just thought we wouldn't do these anymore ^^

Reply:
I'd say we write event migrations for InterdimensionalSiblings in a separate PR, both for node variation and move